getting-started/templates/systemlink-values.yaml

```diff
@@ -92,11 +92,39 @@ global:
     install: false
   ##
   ## Specify secrets containing trusted certificates that should be installed in all the necessary applications.
-  ## The secrets must be deployed manually prior to installing this chart.
+  ## The secrets must be deployed manually prior to installing this chart. Config maps may be used when it is not possible to use a secret.
   ## Example:
-  ##  - secretName: "S3Certificate"
-  ##    key: cert
+  ##  - secretName: "ca-secret"
+  ##    key: "cert"
+  ##  - secretName: "ca-config-map"
+  ##    key: "my-cert"
+  ##    isConfigMap: true
+  ##
+  #  <ATTENTION> - If one or more custom Certificate Authorities are required for HTTPS access to this application
+  #                or for TLS access to any external resources referenced by this application (databases, file storage, etc.),
+  #                a copy of the certificate chain must be deployed as a secret to the cluster prior to deploying this application.
+  #                For certificate authorities, only the public certificate needs to be deployed to the cluster.
+  #
+  #                EXAMPLE:  kubectl -n systemlink-namespace create secret generic my-ca-certificate --from-file=cert=path/to/my-ca.crt
+  #
+  #                Use the 'trustedCertificatesSecrets', 'apiHostCertificateSecret' and 'deployApiHostCertificateToSystems'
+  #                values below to configure the application and registered client systems with the required trust chain.
+  #
   trustedCertificatesSecrets: []
+  ##
+  ## Secret containing the TLS certificate for the API host. This configuration is only necessary when the API
+  ## host is using a custom root CA certificate. This certificate will be deployed and installed on all managed clients
+  ## to enable the communication with the API host. Only valid if deployApiHostCertificateToSystems is true.
+  ##
+  apiHostCertificateSecret:
+  #  secretName: "ca-secret"
+  #  key: "certificate"
+  ##
+  ## Deploy the API host CA root certificate to all managed systems. This configuration is necessary when the API host
```

Comment:
I wonder if we can just leave out these last three lines. They are basically describing again what was described and referenced just above. I think leaving this out could make these docs more clear and readable.